snippets/csharp/getting_started/ClassLibraryProjects/StringLibraryTest/StringLibraryTest.csproj

```
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```

Version 16.4.0 was released.


```suggestion
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
```